### DISCUSSION TITLE
Fresh SO install, trouble getting sysmon events to index from winlogbeat oss 7.13.4

### DISCUSSION THREAD

**Discussion by Twide** [Security-Onion-Solutions/securityonion#5380] [Unsupported Versions]
Hello
I am trying to index sysmon events from winlogbeat, but specifically sysmon events does not index.

-Using latest SO 2.3.70
-Using Winlogbeat OSS 7.13.4 (provided from the soc console)
-Winlogbeat outputs all windows events to SO.
-Sysmon events is being generated at the client and seen by winlogbeat and it's tested with file output, see example below.
-No tagging inside winlogbeat.yml
-Minimal winlogbeat config for windows events:

```
# ======================== Winlogbeat specific options =========================
winlogbeat.event_logs:
  - name: Microsoft-Windows-Sysmon/Operational
```

-My guess is that the event is being lost inside SO.
-It worked just fine with older winlogbeat 6.7xxx and the winlogbeat sysmon processor.

What am i missing?

Example of a sysmon event being generated and output as a file:
`
{"@timestamp":"2021-09-02T09:52:35.198Z","@metadata":{"beat":"winlogbeat","type":"_doc","version":"7.13.4"},"agent":{"ephemeral_id":"c17edf85-4558-46c6-a2f9-8405b507e7a1","id":"19fca540-a985-4c73-8d18-4997841ace01","name":"DESKTOP-JT3TPMB","type":"winlogbeat","version":"7.13.4","hostname":"DESKTOP-JT3TPMB"},"event":{"provider":"Microsoft-Windows-Sysmon","action":"Process Create (rule: ProcessCreate)","created":"2021-09-02T09:52:37.074Z","code":"1","kind":"event"},"log":{"level":"information"},"message":"Process Create:\nRuleName: technique_id=T1059,technique_name=Command-Line Interface\nUtcTime: 2021-09-02 09:52:35.196\nProcessGuid: {1c6f0c84-9ee3-6130-6002-000000001100}\nProcessId: 10156\nImage: C:\\Windows\\System32\\attrib.exe\nFileVersion: 10.0.19041.1 (WinBuild.160101.0800)\nDescription: Attribute Utility\nProduct: Microsoft® Windows® Operating System\nCompany: Microsoft Corporation\nOriginalFileName: ATTRIB.EXE\nCommandLine: attrib\nCurrentDirectory: C:\\ProgramData\\Elastic\\Beats\\winlogbeat\\\nUser: DESKTOP-JT3TPMB\\Anders\nLogonGuid: {1c6f0c84-83bc-6130-257e-020000000000}\nLogonId: 0x27E25\nTerminalSessionId: 1\nIntegrityLevel: High\nHashes: SHA1=B8337FA20077A2C6DB7D01CCC18E328A91255745,MD5=5037D8E6670EF1D89FB6AD435F12A9FD,SHA256=1043111FF07814B0D3439561B4CEE5D7EC1799A7A0A419949FEE707666F6DDED,IMPHASH=2CB38FE7D8F223D9DA50B7CBA9B95A6D\nParentProcessGuid: {1c6f0c84-8d29-6130-e601-000000001100}\nParentProcessId: 440\nParentImage: C:\\Windows\\System32\\cmd.exe\nParentCommandLine: \"C:\\Windows\\system32\\cmd.exe\" ","host":{"mac":["00:0c:29:7b:65:38","02:00:4c:4f:4f:50","00:0c:29:7b:65:42","9c:b6:d0:f8:8a:b8"],"name":"DESKTOP-JT3TPMB","hostname":"DESKTOP-JT3TPMB","architecture":"x86_64","os":{"platform":"windows","version":"10.0","family":"windows","name":"Windows 10 Pro","kernel":"10.0.19041.388 (WinBuild.160101.0800)","build":"19041.388","type":"windows"},"id":"1c6f0c84-f48e-4709-83c0-ea7981ff20ff","ip":["fe80::1c30:a74d:facb:92b5","192.168.23.131","fe80::a4a6:822b:f28a:6792","169.254.103.146","fe80::f810:194e:b8e9:35c9","192.168.23.100","fe80::d081:54e6:26f6:1969","169.254.25.105"]},"winlog":{"channel":"Microsoft-Windows-Sysmon/Operational","version":5,"user":{"domain":"NT AUTHORITY","name":"SYSTEM","type":"User","identifier":"S-1-5-18"},"task":"Process Create (rule: ProcessCreate)","provider_name":"Microsoft-Windows-Sysmon","process":{"pid":5252,"thread":{"id":9120}},"api":"wineventlog","event_id":"1","record_id":48746,"computer_name":"DESKTOP-JT3TPMB","opcode":"Info","provider_guid":"{5770385f-c22a-43e0-bf4c-06f5698ffbd9}","event_data":{"OriginalFileName":"ATTRIB.EXE","LogonGuid":"{1c6f0c84-83bc-6130-257e-020000000000}","User":"DESKTOP-JT3TPMB\\Anders","Company":"Microsoft Corporation","CurrentDirectory":"C:\\ProgramData\\Elastic\\Beats\\winlogbeat\\","ParentCommandLine":"\"C:\\Windows\\system32\\cmd.exe\" ","LogonId":"0x27e25","CommandLine":"attrib","UtcTime":"2021-09-02 09:52:35.196","ProcessId":"10156","Image":"C:\\Windows\\System32\\attrib.exe","Product":"Microsoft® Windows® Operating System","ParentProcessGuid":"{1c6f0c84-8d29-6130-e601-000000001100}","ProcessGuid":"{1c6f0c84-9ee3-6130-6002-000000001100}","Hashes":"SHA1=B8337FA20077A2C6DB7D01CCC18E328A91255745,MD5=5037D8E6670EF1D89FB6AD435F12A9FD,SHA256=1043111FF07814B0D3439561B4CEE5D7EC1799A7A0A419949FEE707666F6DDED,IMPHASH=2CB38FE7D8F223D9DA50B7CBA9B95A6D","RuleName":"technique_id=T1059,technique_name=Command-Line Interface","ParentProcessId":"440","IntegrityLevel":"High","FileVersion":"10.0.19041.1 (WinBuild.160101.0800)","Description":"Attribute Utility","TerminalSessionId":"1","ParentImage":"C:\\Windows\\System32\\cmd.exe"}},"ecs":{"version":"1.9.0"}}`

**Comment by defensivedepth** [comment]
` -It worked just fine with older winlogbeat 6.7xxx and the winlogbeat sysmon processor.`

Are you saying that you used to have the non-FOSS WLB installed and working with Security Onion?

Does the WLB log say that it is successfully connecting to SO?

Also check out the Logstash log... Tail `/opt/so/log/logstash.log` and then restart the WLB service.
